### PR TITLE
CASMTRIAGE-5184 Adding section in upgrade for the nexus upload

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
# Description

This was put into rebuild but not referenced in the upgrade section

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by dzou-hpe — APPROVED** [review]

File: upgrade/Stage_1.md [hunk 1]
```diff
@@ -5,8 +5,9 @@
 
 - [Stage 1 - Ceph image upgrade](#stage-1---ceph-image-upgrade)
   - [Start typescript](#start-typescript)
-  - [Run Ceph Latency Repair Script](#run-ceph-latency-repair-script)
+    - [Run Ceph Latency Repair Script](#run-ceph-latency-repair-script)
```
Comment (leliasen-hpe) [inline upgrade/Stage_1.md:8]: Space added here. I think the space should be removed.

**Review by leliasen-hpe — CHANGES_REQUESTED** [review]

File: upgrade/Stage_1.md (hunk 1 above)
Comment (leliasen-hpe) [inline upgrade/Stage_1.md:8]:
```suggestion
   - [Run Ceph Latency Repair Script](#run-ceph-latency-repair-script)
```

**Review by rnoska-hpe — APPROVED** [review]

**Review by leliasen-hpe — APPROVED** [review]

**Review by djlapenta — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**